components/org.wso2.carbon.identity.api.server.dcr/src/gen/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/dto/ApplicationDTO.java

```diff
@@ -33,6 +33,24 @@ public class ApplicationDTO  {
   
   
   private String clientName = null;
+  private String jwksUri = null;
+  private String tokenEndpointAuthMethod = null;
+  private String tokenEndpointAuthSigningAlg = null;
+  private String sectorIdentifierUri = null;
+  private String idTokenSignedResponseAlg = null;
+  private String idTokenEncryptedResponseAlg = null;
+  private String idTokenEncryptedResponseEnc = null;
```

Comment:
this variable name and `authorizationEncryptedResponseEnc` variable name is bit confusing as they are not self explantory due to having encryption twice in the name. this reads as  "idToken Encrypted Response Encryption"(not sure whether last enc was for encryption). Lets change both variable names to give more clarity.